src/celeritas/inp/Model.hh

```diff
@@ -0,0 +1,33 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/inp/Model.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <string>
+#include <variant>
+
+class G4VPhysicalVolume;
+
+namespace celeritas
+{
+namespace inp
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Set up geometry/material model.
+ */
+struct Model
```

Comment:
`GeometryModel` or `GeoModel`?

Reply:
It has material and region properties too, so I'd say it's more than just the geometry

Reply:
Thought that too, but also was feeling odd to call the geometry (and materials/regions) `model`, even though I don't have yet a better name yet...

Reply:
That's what we ended up calling it in Omnibus 🤷‍♂️